### PR TITLE
upgrades to go 1.16 and embeds json files into binary

### PR DESCRIPTION
**Pull request by danpilch**
Upgrades go to `1.16` and uses new `embed` to add `data/*.json` files into the binary.

### CODE REVIEW

File: nameifier.go [hunk 1]
```diff
@@ -52,8 +45,8 @@ func NewNameifier(basePath string) *Nameifier {
 	n := &Nameifier{
 		BasePath: basePath,
 	}
-	loadJson(n.BasePath+"/"+NounsFilename, n)
-	loadJson(n.BasePath+"/"+AdjectivesFile, n)
+	loadJson(nouns, n)
+	loadJson(adjectives, n)
```
Comment (relistan) [inline nameifier.go:41]: if `BasePath` isn't used for anything else, can we remove it, too?

File: nameifier.go [hunk 2]
```diff
@@ -14,6 +13,11 @@ const (
 	AdjectivesFile = "data/adjectives.json"
```
Comment (relistan) [inline nameifier.go]: Can we remove these if there aren't used any more?

File: nameifier.go (hunk 1 above)
Comment (danpilch) [reply nameifier.go:41]: yeah, you're right. I'll remove it.

File: nameifier.go (hunk 2 above)
Comment (danpilch) [reply nameifier.go]: yep

**Pull request merged**